### PR TITLE
QACI-376 Add test for HTTP Vs Zulu 8.46

### PR DESCRIPTION
**Pull request by AlanRoth**
## Description
Simple test that deploys a servlet and does a POST request, expecting a status code that isn't 500.
### Blockers
This test fails on JDK8 5.2020.3+ with a status code 500, passes on 5.2020.2 JDK8 with a 405. Spawned FISH-515

The test passes on JDK11 5.2020.3+ with a 200.

### New tests
New HttpSupportTest

### Testing Performed
Payara Samples

### Testing Environment
Tested on Zulu 8_252 and Zulu 11.0.1, 5.2020.2, 5.2020.3, 5.2020.5-SNAPSHOT

## Notes for Reviewers
[qaci-376.zip](https://github.com/payara/Payara/files/5316588/qaci-376.zip)
To use the original issue reproducer;
- unzip qaci-376.zip
- mvn clean package
- cp target/qaci-376.war src/main/docker
- docker build -t qaci-376 src/main/docker
- docker run -d --name qaci-376 -p 8080:8080 qaci-376
- wait a few seconds until server is up
- curl http://localhost:8080/qaci-376/hello → status 500
- docker rm -f qaci-376

### CODE REVIEW

**Comment by AlanRoth** [comment]
jenkins test please

**Review by MarkWareham — APPROVED** [review]

**Comment by MarkWareham** [comment]
jenkins test please

**Review by rdebusscher — APPROVED** [review]

**Comment by MattGill98** [comment]
jenkins test please

**Review by lprimak — APPROVED** [review]

**Comment by AlanRoth** [comment]
Still waiting for anybody to pick up FISH-515 to make this test case pass

File: appserver/tests/payara-samples/pom.xml [hunk 1]
```diff
@@ -159,6 +159,12 @@
                 <version>${payara.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>fish.payara.server.internal.common</groupId>
+                <artifactId>common-util</artifactId>
+                <version>${payara.version}</version>
```
Comment (lprimak) [inline appserver/tests/payara-samples/pom.xml:165]: project.version instead of payara.version

**Review by lprimak — CHANGES_REQUESTED** [review]
Changes in line with other test (please wait for that)

**Comment by lprimak** [comment]
jenkins test

**Review by lprimak — APPROVED** [review]

**Pull request merged**